### PR TITLE
:art: Add branded colors "emerald_xxx"

### LINKED ISSUE #25: Add emerald Colors
The colors should be referenced as emerald colors

The colors were defined as the following
image

The colors should be a public resource

### PR DESCRIPTION
**Pull request by Ekt0s**
#### What this PR does?
It adds colors with branded names

#### How should it be manually tested?
Try to use colors in another project using the aar library 

It resolves #25

### CODE REVIEW

File: emeraldcomponents/src/main/res/values/colors.xml [hunk 1]
```diff
@@ -35,4 +35,24 @@
     <color name="emerald_date_filter_green">#95C93D</color>
 
     <color name="emerald_date_view_text">#2d3844</color>
+
+    <!--Bascis color -->
+    <color name="emerald_primary">#95C93D</color>
```
Comment (renanrbs) [inline emeraldcomponents/src/main/res/values/colors.xml]: Minor typo on the comment 😄

File: emeraldcomponents/src/main/res/values/colors.xml [hunk 2]
```diff
@@ -35,4 +35,24 @@
     <color name="emerald_date_filter_green">#95C93D</color>
 
     <color name="emerald_date_view_text">#2d3844</color>
+
+    <!--Bascis color -->
```
Comment (diegolucasb) [inline emeraldcomponents/src/main/res/values/colors.xml]: Typo

File: emeraldcomponents/src/main/res/values/colors.xml [hunk 3]
```diff
@@ -35,4 +35,24 @@
...
+    <public name="emerald_white_2">#FAFAFC</public>
+    <public name="emerald_white_3">#EBEBF0</public>
+    <public name="emerald_white_4">#C3C8D2</public>
+    <public name="emerald_white_5">#96A0AF</public>
+
+    <public name="emerald_black_1">#0C1219</public>
+    <public name="emerald_black_2">#181C24</public>
+    <public name="emerald_black_3">#242932</public>
+    <public name="emerald_black_4">#2D3844</public>
+    <public name="emerald_black_5">#424B54</public>
```
Comment (diegolucasb) [inline emeraldcomponents/src/main/res/values/colors.xml]: It doesn't make the resource public. You should add an existed color resources to `res/public.xml` file

Comment (renanrbs) [reply emeraldcomponents/src/main/res/values/colors.xml]: It is not recognized as a color if the `color` tag is removed, the `public ` tag should be on [public.xml](https://github.com/stone-payments/emerald-components-android/blob/master/emeraldcomponents/src/main/res/values/public.xml)

**Comment by renanrbs** [comment]
Closing this PR due to inactivity

**Pull request closed without merging**